src/koheesio/pandas/categorical_encoding.py

```diff
@@ -0,0 +1,57 @@
+from sklearn.preprocessing import OneHotEncoder
+import pandas as pd
+from pydantic import BaseModel
+from typing import List, Optional
+from koheesio.steps import Step
+
+from typing import List, Dict
+import pandas as pd
```

Comment:
import pandas through the koheesio module (as stated above) to avoid conflict:
`from koheesio.pandas import pandas as pd`

Please run isort (through `make fmt` or `hatch fmt,` or run `ruff`)